### DISCUSSION TITLE
AWIDO Online fails

### DISCUSSION THREAD

**Discussion by Github-Michel** [mampfes/hacs_waste_collection_schedule#1555] [Q&A] [answered]
Hi.
My installation went fine, my city here in Germany is supported by the supplier AWIDO Online. I entered customer and city, but unfortunately the entity in HA always shows up as "off". I got it running once for a short time, when the calendar also showed all the collection dates, but then it went "off" again.
Awido Online shows up in the calendar, but no results as the entity is "off"...
What can be the root cause here?
BR
Michael

**Comment by 5ila5** [comment]
The state of the calendar entity is off when the event is not right now (only past or future collection none due today) but it should have more attributes like start/end time/ message for the next event due. You should also see the collections in your calendar. If you want entities showing a message or a days till.. you should use sensors: [documentation](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/installation.md#configuring-waste-collection-schedules)

If you do not see any results: are there logs showing an error? Have you tried the example shown in the md file?

**Reply by Github-Michel** [reply, marked as answer]
Ok, problem solved :)
In the documentation street and house no. were mentioned as optional ... but I had to delete those lines completely to get it running.
